websites/A/AnimeDao/dist/metadata.json

```diff
@@ -8,10 +8,11 @@
     "en": "Watch anime episodes and movies with English subtitles in SD and HD on any device: pc, tablet, mobile.",
     "pt_BR": "Assista episódios de anime e filmes legendados em inglês, SD ou HD e em qualquer dispositivo: pc, tablet, mobile.",
     "zh_CN": "在任何设备上（PC，平板电脑，移动设备）观看带有英语字幕的高清动漫。",
-    "ga_IE": "Féach "
+    "ga_IE": "Féach ",
```

Comment:
Please delete that translation for us since that doesnt seem correct, I know its not related to the dutch one but thats not good..